src/app.ts

```diff
@@ -146,4 +189,57 @@ export default (
       method: 'deleteSecret',
     });
   });
+
+  app.on("issues", async(context: any) => {
+    const issueComment = context.issue({
+           body: "Thanks for submitting onboarding request!",
+         });
+    
+    // try {
+    //     const data = await parse(context);
+    // } catch {
+    //     app.log.info('Issue was not created using Issue form template (the YAML ones)');
+    //     console.log("We in here but failed");
+    // }
+    
+    const data = { //test data until parse working
+      targetCluster: ["cluster"],
+      teamName: "name",
+      desiredProjectName: "namespace",
+      projectDescription: "description of project",
+      usersNeedingAccess: "users",
+      namespaceQuota: ["custom"],
+      customQuota: "custTest",
+      GPGKey: ""
+    };
+
+    var cluster:string;
+    if (data["targetCluster"][0] == "smaug" || data["targetCluster"][0] == "infra"){
+      cluster = "moc/" + data["targetCluster"];
+    } else{
+      cluster = "emea/" + data["targetCluster"];
+    };
```

Comment:
We want the properly formatted data to come directly from the issue template. No more variable magic should be needed here. Instead you should adjust the issue template to return you "moc/smaug" string directly instead of "smaug" (similarly for all other clusters)